src/app/projects/project-list/project-list-item.component.ts

```diff
@@ -29,9 +29,13 @@ export class ProjectListItemComponent implements OnChanges {
 
   ngOnChanges(changes: NgChanges<ProjectListItemComponent>): void {
     if (changes.project && this.project) {
-      this.providerHostname = this.project.provider?.url
-        ? new URL(this.project.provider.url).hostname
-        : null;
+      try {
+        this.providerHostname = this.project.provider?.url
+          ? new URL(this.project.provider.url).hostname
+          : null;
+      } catch (e) {
+        this.providerHostname = null;
+      }
```

Comment:
This seems to come from #6 . Suggest marking that PR as "Ready for Review" or add "Closes #6" on this PR

Reply:
I have a cool idea for resolution on this- i think input can neatly be sanitised for this purpose with a custom validator. Will continue this change in #6 and will therefore drop it on this PR.